core/src/elements/ons-back-button.js

```diff
@@ -74,15 +74,17 @@ class BackButtonElement extends BaseElement {
    *  [ja]バックボタンの見た目を指定します。[/ja]
    */
 
-  createdCallback() {
-    contentReady(this, () => {
-      if (!this.hasAttribute('_compiled')) {
-        this._compile();
-      }
+  constructor(self) {
+    self = super(self);
+
+    contentReady(self, () => {
+      self._compile();
```

Comment:
Why you are using "self" variable? According to custom elements spec(https://www.w3.org/TR/custom-elements/), you should able to use "this" variable in constructor.